src/common/webpack/config.ts

```diff
@@ -460,7 +461,55 @@ function createStylesRule({
     });
 
     return {
-        test: /\.s?css$/,
+        test: /\.scss$/,
+        sideEffects: isEnvProduction ? true : undefined,
+        use: loaders,
+    };
+}
+
+function createStylesRule({
+    isEnvDevelopment,
+    isEnvProduction,
+    config,
+}: HelperOptions): webpack.RuleSetRule {
+    const loaders: webpack.RuleSetUseItem[] = [];
+
+    if (isEnvProduction) {
+        loaders.push(MiniCSSExtractPlugin.loader);
+    }
+
+    if (isEnvDevelopment) {
+        loaders.push({
+            loader: require.resolve('style-loader'),
+        });
+    }
+
+    loaders.push({
+        loader: require.resolve('css-loader'),
+        options: {
+            esModule: false,
+            sourceMap: !config.disableSourceMapGeneration,
+            importLoaders: 2,
+        },
+    });
+
+    if (!config.transformCssWithLightningCss) {
+        loaders.push({
+            loader: require.resolve('postcss-loader'),
+            options: {
+                sourceMap: !config.disableSourceMapGeneration,
+                postcssOptions: {
+                    config: false,
+                    plugins: [
+                        [require.resolve('postcss-preset-env'), {enableClientSidePolyfills: false}],
+                    ],
+                },
+            },
+        });
+    }
```

Comment:
Let's extract common code to function `getCssLoaders(opts: HelperOptions)` and use it in both functions `createStylesRule` and `createSassStylesRule`